assets/src/edit-story/utils/documentHasSelection.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Checks if the current document has selection,
+ * either by having text selected or being focused on a text/number input.
+ *
+ * @return {boolean} If selection found.
+ */
+function documentHasSelection() {
```

Comment:
The name is not quite representative. I think the main goal here: we expect the native copy/paste to do the job, right? Maybe `nativeCopyPasteExpected`?